test/unit/TestPANOSDriver.py

```diff
@@ -0,0 +1,121 @@
+# Copyright 2016 Dravetech AB. All rights reserved.
+#
+# The contents of this file are licensed under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with the
+# License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+"""Tests."""
+
+import unittest
+
+from napalm_panos import panos
+from napalm_base import exceptions
+
+
+class TestConfigPANOSDriver(unittest.TestCase):
+    """Group of tests that test Configuration related methods."""
+
+    @classmethod
+    def setUpClass(cls):
```

Comment:
I would rather look for alternatives. If we need to change the internals of the tests for some reason it's going to be a pain.

@GGabriele would it be reasonable to treat `.conf` extensions in the same way as `.xml`? In fact, you only support `XML` config right? You could ignore checking the extension of the file and raise an error when trying to parse the file with XML. The xml parser should be able to tell you if the file is a valid XML file or not (it might still be file with the `xml` extension but with complete garbage in it).